addons/mail/tests/test_mail_composer.py

```diff
@@ -218,6 +218,29 @@ def test_composer_template_recipients_private(self):
 class TestMailComposerRendering(TestMailComposer):
     """ Test rendering and support of various html tweaks in composer """
 
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    @users('employee')
+    def test_mail_mass_mode_compose_with_template_state(self):
+        """Test mass mail status for composition with template."""
+        self.test_record.write({'email': 'test@example.com'})
+        composer = self.env['mail.compose.message'].with_context({
+            'default_model': self.test_record._name,
+            'default_composition_mode': 'mass_mail',
+            'active_ids': [self.test_record.id],
+            'active_model': self.test_record._name,
+            'active_id': self.test_record.id
+        }).create({
+            'body': self.body_html,
+            'composition_mode': 'mass_mail',
+            'template_id': self.mail_template.id,
+        })
+
+        with self.mock_mail_gateway(mail_unlink_sent=False):
+            composer._action_send_mail()
```

Comment:
Other tests here seem to specifically call the "prepare_mail_values" method, which seems sufficient here too

Thought of `test_mail_composer_wtpl_recipients` but it's quite massive already so let's keep it simple as you've done ^^